src/ext/dispatch_compat_php7.c

```diff
@@ -123,4 +123,38 @@ void ddtrace_forward_call(zend_execute_data *execute_data, zval *return_value TS
 
     zval_ptr_dtor(&fname);
 }
+
+int ddtrace_should_trace_call(zend_execute_data *execute_data, zend_function **fbc,
```

Comment:
Same comment as in php5 counterpart: Nice self contained function!
Consider using BOOL_T since this is expected to use boolean return values only.